src/go/rpk/pkg/cli/cmd/acl/user.go

```diff
@@ -20,8 +21,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type userCollection struct {
```

Comment:
Same comment as above in the ACLs: why a struct vs. a direct array?

Adding: if we use a struct, at this point I think we may want a consistent envelope type. But I'm not sure an envelope is useful here... undecided.

Reply:
As mentioned on other spots, this was me trying to make sure the return output is Key => value in all cases instead of just returning an array here, key => value hashes/dictionaries elsewhere, etc. Happy to change if you prefer though.